diskfs.go

```diff
@@ -240,24 +238,32 @@ func Open(device string, opts ...OpenOpt) (*disk.Disk, error) {
 	return initDisk(f, ReadWriteExclusive, opt.sectorSize)
 }
 
+// Open a Disk using provided fs.File to a device in read-only mode
+// Use OpenOpt to control options, such as sector size or open mode.
+func OpenStorage(f fs.File, opts ...OpenOpt) (*disk.Disk, error) {
```

Comment:
> OpenStorage is what you advised me during some chat

Yeah, well, I never said I had all the right answers! 😁 

Working it through with you has made some of this clearer.

You are right, ideally it would be `Open(f fs.File)`, and then you would have a convenience function like `OpenFile(device string)` (or `OpenPath()`, etc.). But I am hesitant to break backwards-compatibility unless it brings a _lot_ of benefit.

Well, everything else became `Backend`, so maybe `OpenBackend()`?

Reply:
Let it be `OpenBackend` for now, okay!

Reply:
> Yeah, well, I never said I had all the right answers! 😁

Absolutely! ))

I trust the process and believe, we'll work out the best solution together, iteratively!